### PR TITLE
Use tag instead of year from hodis

### PR DESCRIPTION
**Pull request by mathiasmagnusson**
Stopped using the `year` field from hodis and instead only using `tag`.

Also, if `year` was missing from a user in hodis, we did some really weird stuff around that, which i removed.

### CODE REVIEW

**Comment by mathiasmagnusson** [comment]
Also, if a year was missing from hodis, we used data from the user's input, though that input was (maybe?) removed, so that should not have done anything. So it basically seemed like an unnecessary write access to hodis.

**Comment by mathiasmagnusson** [comment]
This means that new users will have e.g. `D23` instead of `2023`, but old users will still have the old year format. I can fix that in the database, though I'm not sure if it's used that much.

File: resources/views/nominate.blade.php [hunk 1]
```diff
@@ -82,7 +82,7 @@
         <div class="input">
             {!! Form::text('name', NULL, array('placeholder' => 'Namn', 'id' => 'name')) !!}
             {!! Form::text('email', NULL, array('placeholder' => 'KTH-mejladress', 'id' => 'email')) !!}
-            {!! Form::text('year', NULL, array('placeholder' => 'Årskurs', 'id' => 'year', 'class' => 'small')) !!}
+            <p id="year"></p>
```
Comment (Herkarl) [inline resources/views/nominate.blade.php:85]: i assume this is going to be filled by the `autoComplete` function?

**Review by Herkarl — APPROVED** [review]
lgtm

**Pull request merged**